src/elements/element.line.js

```diff
@@ -169,7 +169,7 @@ function useFastPath(vm, options) {
 
 function angleBetween(a, s, e) {
 	return a === s || a === e ||
-		(_angleDiffCW(a, s) > _angleDiffCW(a, e) && _angleDiffCW(s, a) < _angleDiffCW(e, a));
+		(_angleDiffPD(a, s) > _angleDiffPD(a, e) && _angleDiffPD(s, a) < _angleDiffPD(e, a));
```

Comment:
It think you might also be able to write as the below. Then you don't need the `_angleDiffPD` function, which probably makes it faster (e.g. normalize called three times instead of eight and removes four subtractions)

```
function angleBetween(angle, start, end) {
  const a = _normalizeAngle(angle);
  const s = _normalizeAngle(start);
  const e = _normalizeAngle(end);
  return s < e ? a >= s && a <= e : a >= e && a <= s;
}
```

Reply:
Looks a lot better. This one can be used in the radial scale too, so Will place it in helpers (also easier to write tests for it)

Reply:
Did not quite work that way. Current one is the best I can come up with: https://jsperf.com/angle-between-implementations/1